pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -225,11 +230,28 @@ func (w *nfdTopologyUpdater) Stop() {
 }
 
 func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneList, scanResponse resourcemonitor.ScanResponse, readKubeletConfig bool) error {
+	ownerRefs := []metav1.OwnerReference{}
+
+	ns, err := w.k8sClient.CoreV1().Namespaces().Get(context.TODO(), w.kubernetesNamespace, metav1.GetOptions{})
+	if err != nil {
+		klog.InfoS("Cannot set NodeResourceTopology owner reference: %w", err)
+	} else {
+		ownerRefs = []metav1.OwnerReference{
```

Comment:
Just FYI:
There is very nice https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#SetOwnerReference but it would require us using client from `controller-runtime` package.

Reply:
@PiotrProkop
Thanks, kept it as is now, will check regarding controller-runtime.